src/main/java/edu/hawaii/its/api/access/UhCasAttributes.java

```diff
@@ -52,36 +52,8 @@ public String getName() {
         return getValue("cn");
     }
 
-    public String getUsername() {
-        return username;
-    }
-
     @Override
-    public String getUid() {
-        List<String> values = uhAttributeMap.get("uid");
-        if (values != null) {
-            // Check expected case first.
-            if (values.size() == 1) {
-                return values.get(0); // We are done.
-            }
-
-            if (values.size() > 1) {
-                // More than one uid in the results.
-                // Try to match up with the username.
-                for (String s : values) {
-                    if (s.equals(getUsername())) {
-                        return s;
-                    }
-                }
-
-                // Couldn't match up username with uid,
-                // so just return first value.
-                return values.get(0); // We are done.
-            }
-        }
-
-        return ""; // Didn't find anything.
-    }
+    public String getUid() { return getValue("uid"); }
```

Comment:
Expand this line so that:
```
public String getUid() {
    return getValue("uid"); 
}
```
IntelliJ usually formats it like that but the actual code is actually expanded, thanks!